components/gallery/GalleryItemTabsPanel/GalleryItemActivityTable.vue

```diff
@@ -154,7 +154,18 @@ const formatPrice = (price) => {
 }
 </script>
 <style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
 .gallery-item-activity-table {
   overflow-y: auto;
+
+  & .o-table__root {
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+
+    @include mobile {
+      padding-top: 0;
+    }
+  }
```

Comment:
hmm .. I would just lower `py-*` wrapper value instead because we will be migrating oruga components to `libs/ui` so I doubt this specific class would work after that

Reply:
py-x does not support conditional styles like `py-5 mobile:pt-0`. Let me try specifying a class for the o-table to avoid dependency on the internal class.